sage_codes/Safe Curve code

```diff
@@ -428,4 +434,45 @@ sage: res
 /// Hexidecimal representation
 (11 : 23E4A96ED9E13AF0600EA21DB1007D9FB6E47BDD233BBBD82EB8E634305E0B1 : 1)
 
-/// Prime order of curve, l
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
This is a result of a non-solved Merge. Choose the version that you want and remove this line.

Reply:
The issue stated above has been addressed and fixed.